lib/iris/plot.py

```diff
@@ -271,54 +271,78 @@ def _invert_yaxis(v_coord, axes=None):
             axes.invert_yaxis()
 
 
-def _check_contiguity_and_bounds(coord, data, abs_tol=1e-4, transpose=False):
+def _check_bounds_contiguity_and_mask(coord, data, atol=None):
     """
-    Check that the discontinuous bounds occur where the data is masked.
+    Checks that any discontiguities in the bounds of the given coordinate only
+    occur where the data is masked.
 
-    If discontinuity occurs but data is masked, raise warning
-    If discontinuity occurs and data is NOT masked, raise error
+    Where a discontinuity occurs the grid matplotlib creates will not be
+    created correctly. This does not matter if the data is masked in that
+    location as this is not plotted.
+
+    If a discontiguity occurs where the data is *not* masked, an error is
+    raised.
 
     Args:
-        coords:
-            Array of the bounds of a 2D coord, of shape (X,Y,4)
-        data:
-            data of the the cube we are plotting
-        abs_tol:
-            tolerance when checking the contiguity
+        coord: (iris.coord.Coord)
+            Coordinate the bounds of which will be checked for contiguity
+        data: (array)
+            Data of the the cube we are plotting
+        atol:
+            Absolute tolerance when checking the contiguity. Defaults to None.
+            If an absolute tolerance is not set, 1D coords are not checked (so
+            as to not introduce a breaking change without a major release) but
+            2D coords are checked with a relative tolerance.
```

Comment:
> but 2D coords are checked with a relative tolerance.

This is not quite clear.  Maybe something like 
"but 2D coords are always checked, by calling `coord._discontiguity_in_bounds` with its default tolerance"
(If I understood it right)